.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Request a feature for the Python client library
+about: Request a feature
+---
+
```

Comment:
❔ Maybe add a bit here suggesting people discuss feature ideas on Discord for the best chance of getting approved?

Reply:
We mention this in the contributing guidelines.

I'd prefer to keep the template clean to reduce the amount of content, which might put people off.

If you think it's important, feel free to add it though!